components/shared/index.js

```diff
@@ -8,6 +8,7 @@ import BREAKPOINTS from './breakpoints';
 import uniqId from './uniqId';
 import * as INPUT_STYLE from './inputStyle';
 import hexToRgba from './hexToRgba';
+import hexToRgb from './hexToRgb';
```

Comment:
we don't need this anymore, if i am correct.

Reply:
you're correct! :wink:

Reply:
Removed